patient-care/webapp/server/methods.js

```diff
@@ -532,4 +532,27 @@ Meteor.methods({
     // wait for future.throw or future.return to be called
     return future.wait();
   },
+  // refresh the cBioPortal data
+  // NOTE: the only security here for now is that they have a MedBook account
+  refreshCBioPortalData(args) {
```

Comment:
Security looks fine (since you do call out that any registered user can run this.) Jobs.insert will initialize status as `waiting` by default so that's fine.  👍